cmd/simulator/go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/ava-labs/subnet-evm/cmd/simulator
 go 1.17
 
 require (
-	github.com/ava-labs/subnet-evm v0.2.6
+	github.com/ava-labs/subnet-evm v0.2.8
```

Comment:
Is it possible to make this depend on the local version of the code instead?

Reply:
It was my first proposal, let me revert :)